runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/sdk/options/DataflowPipelineOptions.java

```diff
@@ -68,7 +68,9 @@
    */
   @Description("The Dataflow job name is used as an idempotence key within the Dataflow service. "
       + "If there is an existing job that is currently active, another active job with the same "
-      + "name will not be able to be created. Defaults to using the ApplicationName-UserName-Date.")
+      + "name will not be able to be created. Defaults to using the ApplicationName-UserName-Date. "
```

Comment:
Maybe "If there's an existing job running with the ..." or "If there's a running with the same name ...". The "an existing and running job" seems a bit odd.